src/main/resources/assets/eki/lang/ja_jp.lang

```diff
@@ -137,15 +137,15 @@ tile.eki.anchor.warning.tooltip=バインドされた場合はこのブロック
 
 
 //GUIS
-container.eki.ticket_vendor.state.success=発券しました！
+container.eki.ticket_vendor.state.success=発券しました！残高：%d
```

Comment:
there's no %d (integer) parameter provided in this message, will cause format error.

Also, withdraw means get the money out of the vendor, money is saved when player successfully tickets a ticket from vendor with specific amount of money. Maybe first time button meaning is right.

Reply:
sorry, my fault, it's not a fatal format error, but its value means how much money you request to be apply to a ticket. Maybe translate from "Value"?